frontend/src/components/Alert/Alert.tsx

```diff
@@ -40,7 +40,7 @@ const Alert = ({
 		<AntDesignAlert
 			{...props}
 			type={type}
-			className={classNames(props.className, styles.alert)}
+			className={clsx(props.className, styles.alert)}
 			closable={closable != null ? closable : true}
 			showIcon
```

Comment:
alerts configure button doesnt seem to work for any alert
<img width="402" alt="image" src="https://user-images.githubusercontent.com/1351531/218531066-1aea869e-5f45-4cb1-a14a-e548f714595c.png">